chapters.md

```diff
@@ -24,35 +24,21 @@ set to `true` and the second with `ChapterFlagHidden` either set to `false` or n
 at all (in which case the default value of the Element applies, which is `false`).
 Since the parent `ChapterAtom` has its `ChapterFlagHidden` set to `true`, all of its
 children `ChapterAtoms` **MUST** also be interpreted as if their `ChapterFlagHidden` is also
-set to `true`. However, if a `Control Track` toggles the parent's `ChapterFlagHidden`
-flag to `false`, then only the parent `ChapterAtom` and its second child `ChapterAtom`
-**MUST** be interpreted as if `ChapterFlagHidden` is set to `false`. The first child
-`ChapterAtom`, which has the `ChapterFlagHidden` flag set to `true`, retains its value
-until its value is toggled to `false` by a `Control Track`.
+set to `true`.
 
 ### Edition Flags
 
 Three `Edition Flags` are defined to describe the behavior of the `EditionEntry Element`:
-`EditionFlagHidden`, `EditionFlagDefault`, and `EditionFlagOrdered`.
-
-
-#### EditionFlagHidden
-
-The `EditionFlagHidden Flag` behaves similarly to the `ChapterFlagHidden Flag`:
-if `EditionFlagHidden` is set to `true`, its `Child ChapterAtoms Elements` **MUST** also
-be interpreted as if their `ChapterFlagHidden` is also set to `true`, regardless
-of their own `ChapterFlagHidden Flags`. If `EditionFlagHidden` is toggled by a
-`Control Track` to `false`, the `ChapterFlagHidden Flags` of the `Child ChapterAtoms Elements`
-**SHALL** determine whether the `ChapterAtom` is hidden or not.
+`EditionFlagDefault`, and `EditionFlagOrdered`.
 
 
 #### EditionFlagDefault
 
 It is **RECOMMENDED** that no more than one `Edition` have an `EditionFlagDefault Flag`
-set to `true`. The first `Edition` with both the `EditionFlagDefault Flag` set to `true`
-and the `EditionFlagHidden Flag` set to `false` is the Default Edition. When all
-`EditionFlagDefault Flags` are set to `false`, then the first `Edition` with the
-`EditionFlagHidden Flag` set to `false` is the Default Edition. The Default Edition
+set to `true`. The first `Edition` with the `EditionFlagDefault Flag` set to `true` 
```

Comment:
Is the trailing space here intentional? Trailing spaces are significant for Markdown (two spaces == forced line break),
though this instance looks more like an accident.